demo/api.min.js

```diff
@@ -2704,7 +2704,7 @@ class BaseInput extends r {
    */
   handleClickClear() {
     this.inputElement.value = "";
-    this.value = undefined;
+    this.value = "";
```

Comment:
I think it's from a change for another work or there should be a change on `base-input.js` for this.